index.html

```diff
@@ -0,0 +1,464 @@
+        window.launchQueue.addConsumer((launchParams) => {
+          const url = launchParams.targetURL;
+          // If the URL is to one of the app sections, updates the app view to
+          // that section without interrupting currently playing music.
+          if (maybeFocusAppSection(url)) {
+            return;
+          }
+          location.href = url;
+        });
+      </pre>
+      <p>
+        A user, already using the music player app to listen to music,
+        activating the "Library" app shortcut will trigger an app launch to
+        /library which gets routed to the existing app instance, enqueued in the
+        page's {{Window/launchQueue}} which, through the assigned
+        {{LaunchConsumer}}, brings the library section of the music player into
+        focus without affecting the current music playback.
+      </p>
+    </section>
+    <section data-link-for="WebAppManifest">
+      <h2>
+        Extension to the Web App Manifest
+      </h2>
+      <p>
+        The following steps are added to the <a data-cite=
+        "!appmanifest#dfn-extension-point">extension point</a> in the steps for
+        <a data-cite="appmanifest#dfn-processing-a-manifest">processing a
+        manifest</a>:
+      </p>
+      <ol>
+        <li>Run the steps for [=processing the launch_handler member=]
+        given [=ordered map=] |json:ordered map| and [=ordered map=]
+        |manifest:ordered map|.
+        </li>
+      </ol>
+      <section>
+        <h3>
+          [=manifest/launch_handler=] member
+        </h3>
+        <p>
+          The <dfn data-dfn-for="manifest">`launch_handler`</dfn> is a dictionary
+          containing configurations for how web app launches should behave.
+        </p>
+        <p class="note">
+          [=manifest/launch_handler=] is a dictionary despite [=manifest/route_to=] being the only member, this is to give room for more members to be added should other types of behaviors be needed in the future.
+        </p>
+        <p>
+          The steps for <dfn>processing the launch_handler member</dfn>, given
+          [=ordered map=] |json:ordered map|, [=ordered map=]
+          |manifest:ordered map|, are as follows:
+        </p>
+        <ol class="algorithm">
+          <li>If |json|["launch_handler"] does not [=map/exist=], return.
+          </li>
+          <li>If the type of |json|["launch_handler"] is not [=ordered map=],
+          return.
+          </li>
+          <li>Set |manifest|["launch_handler"] to a new [=ordered map=].
+          </li>
+          <li>[=Process the `route_to` member=] passing
+          |json|["launch_handler"], |manifest|["launch_handler"].
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          [=manifest/route_to=] member
+        </h3>
+        <p>
+          The <dfn data-dfn-for="manifest">`route_to`</dfn> member of the manifest is a [=string=] or list of [=strings=] that declares
```

Comment:
> This sentence is a bit of a mess. It's trying to explain three different things:
> 
> 1. The syntax (member of manifest, string or list of strings).
> 2. The semantics (declares whether X or Y).
> 3. That it's done according to a route to target.
> 
> In particular, the "according to a route to target" qualifier reads as though it applies to the final "whether or not" but I think it really applies to the whole thing.
> 
> Rewrite into two or three sentences?

Reduced the amount of information being conveyed since the specifics are covered in the route to targets below.

> "The `route_to` member of the manifest is a string or list of strings that specify one or more route to targets. A route to target declares whether web app launches reuse existing app window clients, if any exist, and whether or not to navigate their browsing contet to the launch's targetURL."
> 
> (Note I also added commas around "if any exist" for clarity.)
> 
> Also: "member of the manifest" -- it's actually a member of the launch_handler.

Good catch, fixed.